.travis.yml

```diff
@@ -306,6 +310,12 @@ before_install:
         # this assumes TEST_THEANO runs in the same build as TEST_MATPLOTLIB
         conda install theano;
     fi
+
+  - if [[ "${TEST_SYMENGINE}" == "true" ]]; then
+        conda config --add channels conda-forge;
+        conda config --add channels symengine;
```

Comment:
If not a tag, can we at least specify a git hash?

Sent from my mobile phone.
On Jul 24, 2016 9:18 AM, "Isuru Fernando" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/sympy/sympy/pull/11002#discussion_r71990328>:
>
> > @@ -306,6 +310,12 @@ before_install:
> >          # this assumes TEST_THEANO runs in the same build as TEST_MATPLOTLIB
> >          conda install theano;
> >      fi
> > +
> > +  - if [[ "${TEST_SYMENGINE}" == "true" ]]; then
> > +        conda config --add channels conda-forge;
> > +        conda config --add channels symengine;
>
> SymEngine channel is updated by this,
> https://github.com/symengine/python-symengine-feedstock/tree/symengine-tokens.
> I change the commit number and the bump the build number to update it.
>
> We can use a git tag of symengine and build and use that version from
> conda-forge channel. I've had to make lots of changes to python wrappers
> and therefore I'm not sure we want to make a tag everytime I do a change to
> the python wrappers for sympy compatibility.
>
> We can always do a symengine release for each sympy release.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sympy/sympy/pull/11002/files/ad9ae61edeac795e1bc7a1d4e7ab7d010f42ded4#r71990328>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABQWNB8MJUMaylvq8qCzbJUJaCwZyeKks5qY4I2gaJpZM4IF-EF>
> .
>